pkgs/servers/nextcloud/packages/29.json

```diff
@@ -140,8 +140,8 @@
     ]
   },
   "maps": {
-    "hash": "sha256-FmRhpPRpMnCHkJFaVvQuR6Y7Pd7vpP+tUVih919g/fQ=",
-    "url": "https://github.com/nextcloud/maps/releases/download/v1.4.0-1-nightly/maps-1.4.0-1-nightly.tar.gz",
```

Comment:
When was this updated to a nightly version? I guess nc4nix still wasn't fixed.

Reply:
Or I accidentally used the wrong tool in 4e0cc1ba6197906c189a3818acba8ebf66448fdb.

In any case, I'm sorry for missing this!